openid-4-verifiable-presentations-1_0.md

```diff
@@ -708,6 +726,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+This section provides guidance on processing and security considerations for the `attachments` parameter defined in (#new_parameters).
+
+### Proof of Possession
+
+In some cases, the Verifier may want to include attachments that are bound to a key or require verification of their origin.
+
+It is up to the ecosystem profile to define how proof of possession is conveyed. Verifiers SHOULD use the `nonce` parameter from the Authorization Request in any proof-of-possession calculation to prevent replay attacks. The Wallet MUST validate such proofs if defined by the profile and ignore or reject attachments that fail validation.
```

Comment:
A verifier generated nonce doesn't prevent replay of a verifier generated object? It definitely does something but I think it's closer to binding it to the response.

Reply:
You are right, I changed it to bind it to the transaction. Open for another good approach